plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/IcebergQueryRunner.java

```diff
@@ -237,7 +239,46 @@ public static void main(String[] args)
                     .build();
 
             Thread.sleep(10);
-            Logger log = Logger.get(IcebergMinIoHiveMetastoreQueryRunnerMain.class);
+            Logger log = Logger.get(IcebergMinioHiveMetastoreQueryRunnerMain.class);
+            log.info("======== SERVER STARTED ========");
+            log.info("\n====\n%s\n====", queryRunner.getCoordinator().getBaseUrl());
+        }
+    }
+
+    public static final class IcebergMinioQueryRunnerMain
+    {
+        private IcebergMinioQueryRunnerMain() {}
+
+        public static void main(String[] args)
+                throws Exception
+        {
+            Logging.initialize();
+
+            String bucketName = "test-bucket";
+            Minio minio = Minio.builder().build();
+            minio.start();
+            minio.createBucket(bucketName);
+
+            DistributedQueryRunner queryRunner = IcebergQueryRunner.builder()
+                    .setCoordinatorProperties(Map.of(
+                            "http-server.http.port", "8080"))
+                    .setIcebergProperties(Map.of(
+                            "iceberg.catalog.type", "TESTING_FILE_METASTORE",
+                            "hive.metastore.catalog.dir", "s3://%s/".formatted(bucketName),
```

Comment:
![image](https://user-images.githubusercontent.com/57260417/229796875-a81effa0-012d-46c6-a19d-612e1a315202.png)

Reply:
That's actually required for managed tables: https://github.com/trinodb/trino/blob/c7e8a6d444fd601e225947a28c7fd1ed85a56d78/plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/file/FileHiveMetastore.java#L351-L355

Since Iceberg tables are external, we could in theory put them elsewhere, but by default, they live in the catalog location. The file hive metastore is designed to store metadata as dot files alongside the table data. For Iceberg, they end up in different directories due to using unique names for Iceberg.

I'm open to revisiting this if you think there's a need, but since this is for testing, I doubt it's worthwhile.